R/center.buoyancy.R

```diff
@@ -23,6 +55,49 @@ center.buoyancy <- function(wtr, depths){
 }
 
 
+
+
+#' Calculates the center of buoyancy for multiple temperature profiles.
```

Comment:
Also, for consistency, could you please include the `@title` and `@description` tags? There are a few functions here that use them that way already and I'd like to maintain consistency.

Reply:
See below. Just for clarification. I'm using the [tidy style guide](http://style.tidyverse.org/code-documentation.html#title-and-description).

Reply:
Yes, while the style guide mentions they are not required, I'd prefer to have them in there.

Reply:
Addressed here: https://github.com/boshek/rLakeAnalyzer/commit/2cb07adb456c7544e0f4a6988f7e379270e2285f